tests/Aspire.EndToEnd.Tests/IntegrationServicesFixture.cs

```diff
@@ -80,7 +80,7 @@ public Task DumpComponentLogsAsync(TestResourceNames resource, ITestOutputHelper
     {
         if (resource == TestResourceNames.None)
         {
-            return;
+            return Task.CompletedTask;
```

Comment:
build fix

Reply:
How'd this get in?

Reply:
oooh! conflicting changes in my PRs!! crap! apologies!